### DISCUSSION TITLE
It is possible to use `inherit` value as color for something components?

### DISCUSSION THREAD

**Discussion by 7iomka** [mantinedev/mantine#2196] [Q&A] [answered]
Hi. I want to use `Loader` but I don't want behavior when this component uses primary color as default.
The best is to use `inherit` value of `color` (similar with no value).
Why?
I have a toggle button which have 2 states
```tsx
const resolvedVariant: ButtonProps['variant'] = isActive ? 'filled' : 'outline';

<Button color="pink" variant={resolvedVariant}>
  { isLoading ? <Loader variant="dots" color={} /> : label }
</Button>
```

Each variant of button have different text color and I don't know it, so, I want to use Loader with inherited color.
As from docs, I don't have a way to change it with styles api globally for example.
Help me please :)

**Comment by rtivital** [comment, marked as answer]
Use currentColor – https://codesandbox.io/s/nervous-tree-54ymnt?file=/src/App.tsx

**Reply by 7iomka** [reply]
> Use currentColor – https://codesandbox.io/s/nervous-tree-54ymnt?file=/src/App.tsx

Thanks!
it's a pity that you can't assign this by default (component don't support styles api) - it would be much more expected than use primary color.

**Reply by rtivital** [reply]
You can use default props on MantineProvider to set it
